docs/framework-components/core-components/configuration-store.mdx

```diff
@@ -0,0 +1,273 @@
+
+## Overview
+
+<ComponentOverview></ComponentOverview>
+
+Configuration Store component entries are saved as JSON documents.
+
+Every category of configuration within the database is distinguished by an `Application_UUID`. While these UUIDs typically identify configuration specific to an individual application, they can also represent generic data utilised by multiple applications.
+
+The fundamental design concept for the Configuration Store component stipulates that any entry saved under a specific `Application_UUID` **MUST** adhere to a schema, ensuring that consuming applications can effectively utilise it. Additionally, it is feasible to store these schemas themselves directly in the component, enabling validation and allowing a universal editing application to generate accurate configuration entries.
+
+Within the Configuration Store component, an entry can be stored for any object identified by a UUID specific to a given application. The various object types represented by UUIDs can be configured by the administrator, initially including `Applications`, `Devices`, and `Schemas`.
+
+- **Applications**, as defined above, can store global defaults using their respective `Application_UUID`.
+- **Devices** correspond to Sparkplug Devices; refer to the [Directory component specification](/docs/framework-components/core-components/directory) for information on locating a device's `Instance_UUID`.
+- **Schemas** pertain to Factory+ schemas as outlined in the [Schema specification](/docs/schemas) and are identified by the schema's `Schema_UUID`.
+
+## Application Schema Requirements
+
+* Application schemas **MUST** use the [2020-12](https://json-schema.org/specification-links.html#2020-12) draft version of JSON schema, i.e. the schema itself should have the `"$schema": "https://json-schema.org/draft/2020-12/schema"` top-level property.
+* Schemas **MUST** include an `$id` field with the value `urn:uuid:{Application_UUID}`, (e.g. `"$id": "urn:uuid:64a8bfa9-7772-45c4-9d1a-9e6290690957"`).
+* Schemas should also include a `title` and `description` for the whole schema, and a `description` for every property, as this information may be used in future to build an editing interface for the config information.
+* It is advisable to include `"additionalProperties": false` on object definitions otherwise the schema will allow random extra properties. This will make extending the schema in the future without invalidating existing data much harder.
+
+## Well-Known UUIDs
+
+Some UUIDs have been reserved that are required for core Factory+ functionality. These UUIDS **MUST** be present and correct in all Factory+ deployments.
+
+### Applications
+
+<WellKnownUUID type='Application' name='General Object Information' uuid='64a8bfa9-7772-45c4-9d1a-9e6290690957' description='🚨 Complete description 🚨'></WellKnownUUID>
```

Comment:
@amrc-benmorrow Could you pop a description into all places with '🚨 Complete description 🚨' in this file please?

Reply:
You can checkout `initial` and run `npm run start` to start a local copy and check changes.